src/App.tsx

```diff
@@ -20,6 +21,33 @@ const App = (props: any) => {
     useEffect(() => {
         if (props.location.search !== '') {
             const query = queryString.parse(props.location.search);
+
+            if (sessionStorage.getItem('nav.klage.klageId') && query && !query.klageid) {
+                if (query.tema) {
+                    let cachedTema = sessionStorage.getItem('nav.klage.tema');
+                    let cachedYtelse = sessionStorage.getItem('nav.klage.ytelse');
+                    if (cachedTema === query.tema) {
+                        if (
+                            cachedYtelse === query.ytelse ||
+                            (query.ytelse === undefined && cachedYtelse === Tema[cachedTema])
```

Comment:
`cachedYtelse === Tema[cachedTema]`, ytelse er det samme som tema? Er det riktig?

Reply:
Ikke helt, her blir ytelsen det lange navnet på temaet. I tilfellet der tema er `FOR` blir da ytelsen `Foreldre- og svangerskapspenger`, oppslag i tema.ts.

Reply:
Det er en fallback som dukker opp et annet sted, og da blir den lagrede ytelsen i storage den her. Denne sjekken sørger for fortsettelse på påbegynt klage dersom query har tema og ikke ytelse.

Reply:
Ok, tror jeg forstår sammenhengen.

Dersom man ikke har en ytelse i query, så må det være en i sessionStorage som matcher temaet (som er i både query og sessionStorage).
Har jeg forstått riktig?

Reply:
Kan ta det litt tidligere i flyten: Om man går inn på siden med kun tema, så blir det satt en ytelse i state og i videresendt klage basert på dette temaet. Den blir da også lagret i `sessionStorage` om man ikke gjør seg ferdig.

Om man kommer inn igjen på siden med kun tema og ikke ytelse, så gjøres det her først en sjekk på om tema matcher det i `sessionStorage`, deretter gjøres det en sjekk på om ytelse i `sessionStorage` er laget på måten beskrevet i avsnittet over. Det indikerer i så fall at det er en påbegynt klage, og klienten henter denne fra backend. 

Om noen av disse ikke matcher, så begynner man på en ny klage.